lib/dal/src/func/binding/attribute_argument.rs

```diff
@@ -123,26 +126,4 @@ impl AttributeArgumentBinding {
             attribute_func_input_location,
         })
     }
-    pub fn assemble_attribute_input_location(
```

Comment:
Moved this to live with its friends. The `AttributeArgumentBinding` struct is relatively self-contained domain-wise after this change.